src/BenchmarksApps/TechEmpower/Minimal/Database/Db.cs

```diff
@@ -2,118 +2,117 @@
 using Dapper;
 using Minimal.Models;
 
-namespace Minimal.Database
+namespace Minimal.Database;
+
+public class Db
 {
-    public class Db
+    private static readonly Comparison<Fortune> FortuneSortComparison = (a, b) => string.CompareOrdinal(a.Message, b.Message);
+
+    private static readonly Random _random = Random.Shared;
```

Comment:
Why do we cache this?

Reply:
Not sure, it's not actually part of my change, I only changed it to file-scoped namespace declaration which is what's mucking of the diff.

Reply:
@sebastienros ?

Reply:
Maybe because it's coming from a previous shared implementation with Platform which was not Random.Shared. We can remove this field.